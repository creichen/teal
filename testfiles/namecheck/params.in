fun f(dup_a, dup_a : t, nondup) = return 0;

class C1 [DupT, DupT, NonDupT, DupFunc] (DupFunc, nonDupFunc, dupMember, dupMethod) = {
      var dupMember := 0;
      fun dupMethod() = return 0;
}
