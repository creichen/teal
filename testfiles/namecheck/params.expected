Multiple declarations for symbol 'dup_a' at locations: 1:7-1:11, 1:14-1:22.
Multiple declarations for symbol 'DupT' at locations: 3:11-3:14, 3:17-3:20.
Multiple declarations for symbol 'DupFunc' at locations: 3:32-3:38, 3:42-3:48.
Multiple declarations for symbol 'dupMember' at locations: 3:63-3:71, 4:7-4:25.
Multiple declarations for symbol 'dupMethod' at locations: 3:74-3:82, 5:7-5:33.
